cli/internal/cmd/workspace/workspace.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+/*
+Package workspace provides functions to generate paths relative to the workspace.
```

Comment:
I don't think I get the reason for this package yet. Can't we integrate this in a file handler or struct? Looks like something that should happen in a more automatic way. `PrefixFileHandler` or so...

Reply:
Do I understand you correctly that you would prefer some interface to handle the prefixing?
E.g. `PrefixHandler` would be set up at the start of a command using the workspace flag.
Then every time we want to print a filepath, instead of calling a distinct function, we instead call, e.g. `PrefixHandler.Path(constants.<SomeFilename>)`, which would return the correct path for the user?

Reply:
Implemented a struct to handle filepath prefixing, can you please review?